src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx

```diff
@@ -225,7 +225,11 @@ export class DataSourceAggregatedView extends React.Component<
                   options={items}
                   renderOption={(option) => (
                     <DataSourceItem
-                      className={'dataSourceAggregatedView'}
+                      className={
+                        this.props.displayAllCompatibleDataSources
+                          ? 'dataSourceAggregatedView'
```

Comment:
just checking due to naming. but it seems like on line 242 if `!this.props.displayAllCompatibleDataSources` then it will render the dataSourceAggregatedViewOuiPanel

but in this logic its `this.props.displayAllCompatibleDataSources` adds the `dataSourceAggregatedView` style. Is that accurate?

Reply:
Hi, Rocky, I confirm the logic here is when :
displayAllCompatibleDataSources -> true -> list all ( code: https://github.com/yujin-emma/OpenSearch-Dashboards/blob/main/examples/multiple_data_source_examples/public/components/data_source_list_all_example.tsx#L104 ) 
displayAllCompatibleDataSources -> false -> list active ( code: https://github.com/yujin-emma/OpenSearch-Dashboards/blob/main/examples/multiple_data_source_examples/public/components/data_source_list_active_example.tsx#L111 )